docker-compose.builder.yml

```diff
@@ -0,0 +1,12 @@
+# inspired from https://hackernoon.com/a-better-way-to-develop-node-js-with-docker-cd29d3a0093
+version: '3.7'
+
+services:
+  install:
+    image: node:14
+    volumes:
+      - .:/nuclear/
+    working_dir: /nuclear/
+    command: bash -xc "
+        npm install;
+        npx lerna bootstrap"
```

Comment:
`lerna bootstrap` will run as a postinstall step, so all we need is `npm install`.

Btw, I read that article some time ago, and again now, and I still don't see what we gain by splitting this part into a separate file, as opposed to a service in the same docker-compose file as the rest.